README.md

```diff
@@ -102,3 +102,9 @@ The `make dev` and `make release` commands do three things:
 You can print to the browser console using the standard `print_endline` function. This is probably the easiest method right now.
 
 `js_of_ocaml` does support source maps and has some other flags that might be useful. If you experiment with those and get them to work, please update this README with some notes.
+
+### Testing
+
+You can run all of the unit tests located in `src/hazelcore/test` by running `dune runtest`.
```

Comment:
Let's change `dune runtest` to `make test` since we say to use `make` everywhere else.

Reply:
Oh shoot, I already did! This is just a typo.